packages/dev/core/src/Shaders/ShadersInclude/pbrBlockReflection.fx

```diff
@@ -287,13 +292,9 @@
                 #else
                     environmentIrradiance = computeEnvironmentIrradiance(irradianceVector);
                 #endif
-                
-                #ifdef SS_TRANSLUCENCY
-                    outParams.irradianceVector = irradianceVector;
-                #endif
             #endif
         #elif defined(USEIRRADIANCEMAP)
-            vec4 environmentIrradiance4 = sampleReflection(irradianceSampler, reflectionCoords);
+            vec4 environmentIrradiance4 = sampleReflection(irradianceSampler, irradianceVector);

```

Comment:
The irradiance was incorrectly sampled with the reflection vector instead of the irradiance vector.
I had to reorder some of the shader logic in order to get access to this vector.